README.md

```diff
@@ -218,16 +231,21 @@ Development of `fused-effects` is typically done using `cabal v2-build`:
 
 ```shell
 cabal v2-build # build the library
-cabal v2-test  # build and run the examples, unit tests, and doctests
+cabal v2-test  # build and run the examples and tests
 ```
 
 The package is available on [hackage][], and can be used by adding it to a component’s `build-depends` field in your `.cabal` file.
 
 [hackage]: http://hackage.haskell.org
 
+
+### Testing
+
+`fused-effects` comes with a rigorous test suite. Each law or property stated in the Haddock documentation is checked using generative tests powered by the `hedgehog`. We have a high degree of confidence in the correctness of all the carrier types provided in the `Control.Carrier` hierarchy.
```

Comment:
```suggestion
`fused-effects` comes with a rigorous test suite. Each law or property stated in the Haddock documentation is checked using generative tests powered by `hedgehog`. We have a high degree of confidence in the correctness of all the carrier types provided in the `Control.Carrier` hierarchy.
```

I _really_ don’t want to say anything about having a high degree of confidence in anything, ever.

Reply:
You’re right; that’s just asking for trouble.